src/wp-includes/media.php

```diff
@@ -1911,6 +1918,39 @@ function wp_img_tag_add_loading_attr( $image, $context ) {
 	return $image;
 }
 
+/**
+ * Adds `decoding=async` attribute to an `img` HTML tag.
+ *
+ * @since 6.0.0
+ *
+ * @param string $image         The HTML `img` tag where the attribute should be added.
+ * @param string $context       Additional context to pass to the filters.
+ * @param int    $attachment_id Image attachment ID.
+ * @return string Converted `img` tag with `decoding` attribute added.
+ */
+function wp_img_tag_add_decoding_async_attr( $image, $context, $attachment_id ) {
```

Comment:
```suggestion
 * @param string $image   The HTML `img` tag where the attribute should be added.
 * @param string $context Additional context to pass to the filters.
 * @return string Converted `img` tag with `decoding` attribute added.
 */
function wp_img_tag_add_decoding_async_attr( $image, $context, $attachment_id ) {
```

Reply:
why remove the attachment_id from doc block here?

Reply:
nm, i see that matches the approach of other image transforms that may not have the attachment id. updating